sim/druid/runes.go

```diff
@@ -9,33 +9,80 @@ import (
 )
 
 func (druid *Druid) ApplyRunes() {
+	// Helm
+	druid.applyGaleWinds()
+	druid.applyGore()
+
 	// Chest
 	druid.applyFuryOfStormRage()
-	// druid.applyLivingSeed()
-	// druid.applySurvivalOfTheFittest()
 	druid.applyWildStrikes()
 
+	// Bracers
+	druid.applyElunesFires()
+
 	// Hands
-	// druid.applyLacerate()
 	druid.applyMangle()
-	druid.applySunfire()
-	// druid.applyWildGrowth()
+	druid.registerSunfireSpell()
 
 	// Belt
 	druid.applyBerserk()
 	druid.applyEclipse()
-	// druid.applyNourish()
 
 	// Legs
 	druid.applyStarsurge()
 	druid.applySavageRoar()
-	// druid.applyLifebloom()
-	// druid.applySkullBash()
 
 	// Feet
 	druid.applyDreamstate()
 	druid.applyKingOfTheJungle()
-	// druid.applySurvivalInstincts()
+}
+
+func (druid *Druid) applyGaleWinds() {
+	if !druid.HasRune(proto.DruidRune_RuneHelmGaleWinds) {
+		return
+	}
+
+	druid.RegisterAura(core.Aura{
+		Label:    "Gale Winds",
+		ActionID: core.ActionID{SpellID: int32(proto.DruidRune_RuneHelmGaleWinds)},
+		Duration: core.NeverExpires,
+		OnReset: func(aura *core.Aura, sim *core.Simulation) {
+			aura.Activate(sim)
+		},
+	})
+}
+
+func (druid *Druid) applyGore() {
+	if !druid.HasRune(proto.DruidRune_RuneHelmGore) {
+		return
+	}
+
+	druid.RegisterAura(core.Aura{
+		Label:    "Gore",
+		ActionID: core.ActionID{SpellID: int32(proto.DruidRune_RuneHelmGore)},
+		Duration: core.NeverExpires,
+		OnReset: func(aura *core.Aura, sim *core.Simulation) {
+			aura.Activate(sim)
+		},
+	})
+}
+
+const (
+	Gore_BearResetProcChance = .15
+	Gore_CatResetProcChance  = .05
+)
+
+// TODO: Bear spells not implemented: MangleBear, Swipe, Maul
+func (druid *Druid) rollGoreBearReset(sim *core.Simulation) {
+	if sim.RandomFloat("Gore (Bear)") < Gore_BearResetProcChance {
+		druid.MangleBear.CD.Reset()
+	}
+}
+
+func (druid *Druid) rollGoreCatReset(sim *core.Simulation) {
+	if sim.RandomFloat("Gore (Cat)") < Gore_CatResetProcChance {
+		druid.TigersFury.CD.Reset()
+	}
```

Comment:
I opted to break up Gore and Elune's Fires like this for performance reasons. Originally I gave each one an aura with on `OnSpellHitDealt` with different conditions but that adds a lot of extra conditionals on every single hit and ultimately I think this approach requires a bit more dev work but will be more performant since the relevant code is only being checked when the actual trigger spells hit